rpcs3/Emu/SysCalls/Modules/cellPng.h

```diff
@@ -9,6 +9,13 @@ enum CellPngTxtType : s32
 	CELL_PNG_ITXT = 2,
 };
 
+enum CellPngUnknownLocation : s32
```

Comment:
Original style is usually kept for such things. And enum class will only become a pain in the ass there, not improving anything.